staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -196,6 +196,19 @@ func ConnectResource(connecter rest.Connecter, scope *RequestScope, admit admiss
 			scope.err(err, w, req)
 			return
 		}
+
+		// audit portForwardOption when SPDY
+		audit.LogRequestObject(ae, opts, scope.Kind.GroupVersion(), scope.Resource, scope.Subresource, scope.Serializer)
```

Comment:
at first glance, I think the LogRequestObject line is reasonable. please drop the portforward changes from this PR... more discussion of the approach is needed before implementation

Reply:
also, this needs a test to make sure connect options show up properly in the audit log

Reply:
This is what I am confused about. I don't know if it makes sense to just increase the audit, because I don't see the port information being passed when a `connect` occurs.

Currently only PodForWardOptions{Kind,Apiversion} can be recorded
Or modify the PR to only add the audit when `connect`, as a simple start to audit PodForWardOptions...
I will add a test, if this pr is useful....

Reply:
Is `scope.Kind.GroupVersion()` the correct GV for the audit log?  Is the GV supposed to match that of the overall resource or the GV of `opts`?